machinelearning/pytorch.md

```diff
@@ -5,54 +5,73 @@
 PyTorch is an open-source machine learning framework based on the Torch library, used for applications such as computer vision and natural language processing. It can be used across training and inference of deep neural networks. This document covers how to use PyTorch based machine learning inference on Graviton CPUs, what runtime configurations are important and how to debug any performance issues. The document also covers instructions for source builds and how to enable some of the downstream features.
 
 # How to use PyTorch on Graviton CPUs
-There are multiple levels of software package abstractions available: Python wheel (easiest option but multiple wheels to be installed), AWS DLC (Deep Learning Container, comes with all the packages installed) and the Docker hub images (additionally comes with MLPerf benchmarks). Examples of using each method are below.
+There are multiple levels of software package abstractions available: Python wheel (easiest option to get the release features but multiple wheels to be installed), AWS DLC (Deep Learning Container, comes with additional optimizations on top of wheels, and all the packages installed) and the Docker hub images (additionally comes with downstream experimental features). Examples of using each method are below.
```

Comment:
Lead with DLC and say best perf?

Reply:
Hi @AGSaidi , addressed all your feedback in the next update